package.json

```diff
@@ -61,6 +61,8 @@
   },
   "dependencies": {
     "@gitpod/local-app-api-grpcweb": "main",
+    "@gitpod/gitpod-protocol": "gpl-5862-snapshot",
```

Comment:
we probably can switch to main for gitpod packages, everything should have been merged already

Reply:
done

Reply:
I tried updating them in gitpod-shared but got a compilation error because `PortsStatus.globalPort` doesn't exist

Reply:
I found the PR that removed them https://github.com/gitpod-io/gitpod/pull/6462
Do you want to to fix this in this PR, what should be used instead of `globalPort` or should I just remove it?

Reply:
Global port should be removed